aqi_monitor/apps/monitor/tasks.py

```diff
@@ -17,6 +20,16 @@
 def get_aqi():
     region = "saratov"
     response = requests.get(
-        "https://api.waqi.info/feed/" + region + "/?token=7162b8553552c91543ed0ae44c44e59b3724a08e")
-    result = json.loads(response.text)
-    print(result)
\ No newline at end of file
+        f"https://api.waqi.info/feed/{region}/?token=7162b8553552c91543ed0ae44c44e59b3724a08e")
+    data = json.loads(response.text)["data"]
+    idx = {"id_region": data["idx"]}
+    aqi_val = {"aqi": data["aqi"]}
+    aqi = fun.parse_iaqi(data["iaqi"])
+    message = {"message": fun.out_message(aqi_val["aqi"])}
+    time = {"time": fun.time_convert(data["time"])}
+    result = dict()
+    result.update(aqi_val)
+    result.update(idx)
+    result.update(aqi)
+    result.update(time)
+    result.update(message)
```

Comment:
I suggest to avoid such actions in your code.
Try to reconsider your approach and refactor this.

Reply:
Fixed